scala-core-oop/src/main/scala/com/baeldung/scala/lifting/Examples.scala

```diff
@@ -0,0 +1,58 @@
+package com.baeldung.scala.lifting
+
+import scala.concurrent._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import cats.data.OptionT
+import cats.implicits._
+import cats._
+
+object Examples {
+
+  val squareRoot: PartialFunction[Double, Double] = { case x if x >= 0 => Math.sqrt(x) }
+
+  def getSqrtRootMessagePartialFunction(x: Double) = {
+    if (squareRoot.isDefinedAt(x)) {
+      s"Square root of $x is ${squareRoot(x)}"
+    } else {
+      s"Cannot calculate suqare root for $x"
+    }
+  }
+
+  def getSqrtRootMessageTotalFunction(x: Double) = {
+   squareRoot.lift(x).map(result => s"Square root of ${x} is ${result}")
+                     .getOrElse(s"Cannot calculate suqare root for $x")
```

Comment:
Line continuations should have a two space indent

Reply:
done